README.md

```diff
@@ -103,90 +106,73 @@ These instructions assume you've used the
+5. For ZCash, [zcashd v5.1](https://z.cash/download/).
+6. For Bitcoin Cash, [Bitcoin Cash Node v24+](https://bitcoincashnode.org/en/)
 
 ### Connect Wallets and Register
 
-1. Start the client. `./dexc`
-2. In your browser, navigate to **localhost:5758**
-
-<img src="docs/images/omnibar-client.png" width="250">
-
-3. Create your **client application password**. You will use this password to perform all future security-sensitive client operations, including registering, signing in, and trading.
-
-<img src="docs/images/client-pw.png" width="250">
-
-4. Connect to your Decred wallet. The client will auto-fill most of your wallet settings, but you will need to specify the account name. If you haven't already, follow the instructions above to create a trading account. If you really, really want to trade on the default wallet account, the account name is `default`. Enter the wallet password, which is the password you set up with **dcrwallet**. Enter the app password you created in step 3.
-
-<img src="docs/images/decred-reg.png" width="250">
-
-5. Enter the dex address, probably **dex.decred.org**.
-
-<img src="docs/images/add-dex-reg.png" width="250">
-
-6. Check the registration fee, and enter your password one more time to authorize payment.
-
-<img src="docs/images/confirm-reg.png" width="250">
-
-7. On the **markets view**, while you're waiting for confirmations on your registration fee, add a Bitcoin wallet. You'll need to specify the wallet name. If you haven't already, follow the instructions above to create a trading wallet. If you really, really want to trade on the default wallet, leave the wallet name blank. Enter the wallet password you set up with **bitcoin-cli**. If you really, really want to trade on an unencrypted wallet, you can leave the wallet password blank. Enter the app password you created in step 3.
-
-<img src="docs/images/create-btc.png" width="250">
-
-8. And that's it! Once your registration fee has enough confirmations, you can begin trading.
+1. Start the client. Either go to the "DEX" tab within Decrediton, or with the
+   standalone client, open a command prompt in the folder containing the
+   pre-compiled dexc client files and run `./dexc` (`dexc.exe` on Windows).
+2. In your web browser, navigate to http://localhost:5758. Skip this step if
+   using Decrediton.
+
+[//]: # "TODO: update with a few screenshots for the new steps below"
+
+1. Set your new **client application password**. You will use this password to
+   perform all future security-sensitive client operations, including
+   registering, signing in, and trading.
+
+2. Choose the DEX server you would like to use. Either click one of the
+   pre-defined hosts such as **dex.decred.org**, or enter the address of a known
+   server that you would like to use.
+
+3. The DEX server will support paying the one time setup fee in one or more
+   assets. (This is a nominal amount just to discourage abuse and maintain a
```

Comment:
Maybe `The DEX server will offer a choice of assets with which to pay the one-time setup fee.`